http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -503,14 +506,11 @@ class GKEInterpreter[F[_]: Parallel: ContextShift: Timer](
       )
 
       _ <- if (!params.errorAfterDelete) {
-        for {
-          _ <- authProvider.notifyResourceDeleted(dbApp.app.samResourceId,
-                                                  dbApp.app.auditInfo.creator,
-                                                  params.googleProject)
-          _ <- appQuery.markAsDeleted(dbApp.app.id, ctx.now).transaction
-        } yield ()
+        authProvider
```

Comment:
I feel like we should always do `appQuery.markAsDeleted` and `authProvider.notifyResourceDeleted` together.. can we move the authProvider call too to `LeoPubsubMessageSubscriber`?